### PR TITLE
Conv updates

### PR DESCRIPTION
**Pull request by dverbeir**

### CODE REVIEW

**Comment by GregoryVds** [comment]
Hi David,
Unfortunately I think we need to bump Ubuntu to 20.x in the Dockerfile as 19.04 is EOF.

File: convert.h [hunk 1]
```diff
@@ -38,6 +38,7 @@
 #include <netinet/in.h>
 
 #define CONVERT_VERSION 1
+#define CONVERT_MAGIC_NO 0x2263
```
Comment (matttbe) [inline convert.h:41]: or 8803 in decimal format :)
(like the RFC)
But best to keep 0x2263 as specified on the RFC

**Review by matttbe — APPROVED** [review]
Looks good to me, good job!

**Review by GregoryVds — APPROVED** [review]
Thanks for adding this!

**Review by GregoryVds — APPROVED** [review]

**Review by GregoryVds — APPROVED** [review]

**Review by GregoryVds — APPROVED** [review]

**Comment by GregoryVds** [comment]
I think you need something like this in the Dockerfile:
`ENV DEBIAN_FRONTEND noninteractive`

Docker build is blocked on the following:
```Please select the geographic area in which you live. Subsequent configuration
questions will narrow this down by presenting a list of cities, representing
the time zones in which they are located.
  1. Africa      4. Australia  7. Atlantic  10. Pacific  13. Etc
  2. America     5. Arctic     8. Europe    11. SystemV
  3. Antarctica  6. Asia       9. Indian    12. US
Geographic area: ```

**Comment by GregoryVds** [comment]
> I guess this will fail, e.g. clang-tools-6.0 is not available in 20.04. Can we install any version of CLang?

No idea :-) Let's see what is says with another version :crossed_fingers:

**Comment by GregoryVds** [comment]
:'( New cppcheck is apparently catching new lint errors

**Comment by dverbeir** [comment]
@GregoryVds Can we merge this PR now? I forced usage of EOL'd Ubuntu 19.04 for now as you suggested.

**Comment by GregoryVds** [comment]
@dverbeir Sure, looking good to me!
Sorry, I didn't notice that you pushed that last update on the Dockerfile

**Pull request merged**